client/packages/system/src/Item/api/api.ts

```diff
@@ -112,7 +116,7 @@ export const getItemQueries = (sdk: Sdk, storeId: string) => ({
         key: itemParsers.toSortField(sortBy),
         desc: sortBy.isDesc,
         storeId,
-        filter: filterBy,
+        filter: { ...filterBy, isVisible: true },
```

Comment:
offroad: after the integration tests I had a bunch of empty items in mSupply, which then sync'd back ( empty name & code or Non-Stock items with UUID name & code ).

looking at why, I noticed that the `isVisible` filter is now available and working, so have popped it in here.